<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>充电计费管理员客户端</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="account" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>671</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="l2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>账号管理</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="l4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>身份</string>
    </property>
   </widget>
   <widget class="QLabel" name="l5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="l1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="userState">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLabel" name="userIdentity">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>注销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="info" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>811</width>
     <height>671</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>信息查询与管理</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="queue" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查询排队情况</string>
     </property>
    </widget>
    <widget class="QPushButton" name="queueBtn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="queueTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>361</width>
       <height>171</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>充电桩编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计使用次数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计使用时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计充电电量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计充电费用</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计服务费用</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计费用</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="pileStateSearch" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查询充电桩状态</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pileStateBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="pileStateTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>充电桩编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计使用次数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计充电时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计充电电量</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="pileList" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查询充电桩报表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pileListBtn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="pileListTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>361</width>
       <height>171</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>充电桩编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>天</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>月</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计使用次数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计充电时长</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计充电电量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计充电费用</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计服务费用</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>累计费用</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="pileStateUpdate" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>修改充电桩状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>充电桩编号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>状态</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pileNum">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="switchState">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>运行</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>关机</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>故障</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="updateBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>更新</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>16</width>
     <height>671</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
